css/common.css

```diff
@@ -100,7 +100,6 @@ summary > .oo-ui-labelElement-label {
 
 .showClosed {
 	float: right;
-	margin-top: -6px;
```

Comment:
This negative margin is for the alignment of the button within the warning box, not the box itself.